Cargo.toml

```diff
@@ -11,9 +11,9 @@ repository = "https://github.com/filecoin-project/rust-gpu-tools"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-ocl = { version = "0.19.4", package = "fil-ocl" }
 dirs = "2.0.2"
 sha2 = "0.8.2"
 thiserror = "1.0.10"
 lazy_static = "1.2"
 log = "0.4.11"
+opencl3 = "0.2.1"
```

Comment:
do we need to specify any features here?

Reply:
It worked fine so far, but good point about reducing it to the minimum. Is it OK if we do this in a future iteration?